src/Order.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+const Order = props => {
+  const [orders, setOrders] = useState(0);
+
+  const orderOne = () => {
+    setOrders(orders + 1);
+  };
+
+  return (
+    <li>
+      {props.orderType}: {orders} <RestaurantButton data={orderOne} />
```

Comment:
Do you think `data` is the best name for the prop here? How could we improve it?

Reply:
Hi, 40thieves  thank you for your feedback I have fixed the issue and used meaningful name